lake/commits/actions.go

```diff
@@ -56,15 +57,20 @@ type Commit struct {
 	Author  string      `zed:"author"`
 	Date    nano.Ts     `zed:"date"`
 	Message string      `zed:"message"`
+	Meta    zed.Value   `zed:"meta"`
+}
+
+func (c *Commit) String() string {
+	//XXX need to format Message field for single line
+	return fmt.Sprintf("COMMIT %s -> %s %s %s %s", c.ID, c.Parent, c.Date, c.Author, c.Message)
 }
 
 func (c *Commit) CommitID() ksuid.KSUID {
 	return c.ID
 }
 
-func (c *Commit) String() string {
-	//XXX need to format Message field for single line
-	return fmt.Sprintf("COMMIT %s -> %s %s %s %s", c.ID, c.Parent, c.Date, c.Author, c.Message)
+func (c *Commit) AppMeta() zed.Value {
+	return c.Meta
```

Comment:
These method changes don't appear to be necessary for this PR, so remove them to minimize the diff.

Reply:
Ah, done! I'll double-check diff in future. thanks!